types.go

```diff
@@ -392,12 +392,22 @@ type FlashbotsGetBundleStatsParam struct {
 }
 
 type FlashbotsGetBundleStatsResponse struct {
-	IsSimulated    bool      `json:"isSimulated"`
-	IsSentToMiners bool      `json:"isSentToMiners"`
-	IsHighPriority bool      `json:"isHighPriority"`
-	SimulatedAt    time.Time `json:"simulatedAt"`
-	SubmittedAt    time.Time `json:"submittedAt"`
-	SentToMinersAt time.Time `json:"sentToMinersAt"`
+	IsSimulated            bool                      `json:"isSimulated"`
+	IsSentToMiners         bool                      `json:"isSentToMiners"`
+	IsHighPriority         bool                      `json:"isHighPriority"`
+	SimulatedAt            time.Time                 `json:"simulatedAt"`
+	SubmittedAt            time.Time                 `json:"submittedAt"`
+	SentToMinersAt         time.Time                 `json:"sentToMinersAt"`
+	ConsideredByBuildersAt []*ConsideredByBuildersAt `json:"consideredByBuildersAt"`
+	SealedByBuildersAt     []*SealedByBuildersAt     `json:"sealedByBuildersAt"`
+}
+type ConsideredByBuildersAt struct {
```

Comment:
These 2 types could be a single `BuilderPubleyWithTimestamp`, and be used for both.